src/app/app-routing.module.ts

```diff
@@ -1,8 +1,20 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LandingPageComponent } from './components/landing-page/landing-page.component';
+import { QuestionApprovalPageComponent } from './components/question-approval-page/question-approval-page.component';
+import { PagenotfoundComponent } from './components/pagenotfound/pagenotfound.component';
+import { ActionsComponent } from './components/actions/actions.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+
+  { path: 'actions', component: ActionsComponent },
+  { path: 'actions/approve', component: QuestionApprovalPageComponent },
```

Comment:
It's probably worth putting this under a `/questions` subresource. Reason being that in the future, when someone tries to create a new organisation, we probably want to be involved in that process (i.e. approving the new organisation for users). At that point we'd then have to relocate this feature area to another path.

Reply:
I'll bring up this point later for discussion so I can understand what you mean better.